### PR TITLE
add Omega branch

### PR DESCRIPTION
**Pull request by fuzzard**
@gade01 can you just confirm the yml file is correct

### CODE REVIEW

**Review by gade01 — APPROVED** [review]
@fuzzard It looks great, thanks.

**Review by enen92 — CHANGES_REQUESTED** [review]
Addon checker job is missing on omega (btw, also missing in nexus).
File in matrix: https://github.com/xbmc/repo-resources/blob/matrix/.github/workflows/addon-checker.yml

**Comment by fuzzard** [comment]
Added addon-checker.yml

File: README.md [hunk 1]
```diff
@@ -4,7 +4,45 @@
...
 
-This branch is not used for public. Please use one of the other branches available for submitting add-ons.
+This branch contains a certain category of add-ons from which our back-end script creates .zip files which are made available to each Kodi client.
+* Language add-ons
+* Image resources
+* Controller profiles
+
+## How to submit your add-on and subsequent updates ##
+
+Your add-on must follow our strict repository rules to be considered for inclusion. Please consult the [Add-on rules] (http://kodi.wiki/view/Add-on_Rules) wiki page for further details. Please review these rules carefull before submitting your add-on. Should you have any questions regarding them please start a forum thread in one of the following locations
```
Comment (garbear) [inline README.md]: Spelling: carefull -> carefully

File: README.md [hunk 2]
```diff
@@ -4,7 +4,45 @@
...
+* [Python add-ons] (http://forum.kodi.tv/forumdisplay.php?fid=26)
+* [Skins] (http://forum.kodi.tv/forumdisplay.php?fid=12)
+
+After you have read the repository guidelines and made sure your addon is compliant with them, you may begin the submission process. By forking this repository and creating a pull-request to the correct repository branch you ask permissing to included you add-on to the official Kodi repository. Subsequent updates can be done in a similar way by updating the code and creating a new pull-request again. Make sure that your local git clone is always rebased before send a pull-request.
```
Comment (garbear) [inline README.md]: grammar: "included you add-on to" -> "include your add-on in".

grammar: "send" -> "sending"

File: README.md [hunk 3]
```diff
@@ -4,7 +4,45 @@
...
+
+Keep in mind that add-ons in the official repository should be considered stable. This means that they should be well-tested before you submit them for inclusion. Because they are for stable users, they should avoid being updated too often. Too often is of course subjective. If your add-on is in rapid development, and features are constantly being added, hold off until you have hit a good stopping point and tested the current version.
+This means that you should not submit a request every time you change your code. If you are submitting updates more than once per week something is wrong. Once or twice per month is probably a better goal, barring unforeseen conditions (like a content source changing its paths). With good reasons provided we will of course make exceptions as we strive to prove the best user experience.
+
+## Compatibility
+
+This branch is used for add-ons that are coded for Kodi v21 Omega builds and higher only. From these code repositories and branches our back-end uploades .zip files of the compatible add-ons to our main mirror server.
```
Comment (garbear) [inline README.md]: Spelling: "uploades" -> "uploads"

File: README.md [hunk 4]
```diff
@@ -4,7 +4,45 @@
...
+* [Mirror of Kodi v21 Omega compatible add-ons] (http://mirrors.kodi.tv/addons/omega/)
+
+## Status
+
+* New add-on additions: **Accepted**
+* Updating already present add-ons: **Accepted**
+
+## Disclaimer ##
+
+The contents of this repository mainly consist of add-on created by third party developers. Team Kodi holds no responsibility for it's contents.
```
Comment (garbear) [inline README.md]: Grammar: "add-on" -> "add-ons"

Grammer: "it's" -> "its"

**Review by enen92 — APPROVED** [review]

**Pull request merged**